code/game/objects/structures/tables_racks.dm

```diff
@@ -294,6 +297,59 @@
 	..()
 	return SECONDARY_ATTACK_CONTINUE_CHAIN
 
+/obj/structure/table/CtrlShiftClick(mob/living/user)
```

Comment:
Ctrl+Shift+Click is too clunky. Should be right-click with nothing in your hand.

Then also make sure that you show it in a contextual screentip.

Reply:
right click with nothing in hand is too easy to accidentally do